### DISCUSSION TITLE
Meeting Agenda 3/19

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#335] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Review new things
    1. `/rerun` in pr comment to rerun failed ci jobs
    2. changes to visualization - TreeDisplay
3. Sprint Planning
4. d3 knowledge sharing?
5. discuss poster
6. Meet with Darian

**Comment by moreno-michael** [comment]
Meeting minutes 3/19

1. Task Review
- All issues from previous sprint still in progress
2. Review new things
- `/rerun` functionality in PR's discussed
- TreeDisplay update discussed
- Smooth vs stepped lines? Stepped lines between balancers and monoliths seem easier on the eyes
3. Sprint Planning
- Tasks from previous sprint pushed to this sprint
4. d3 knowledge sharing
5. Expo poster discussion
- Draft showed and discussed
- Brainstormed the types of graphs and associated data we could have on the poster to portray the benefit of our product
- Latency could be a metric but it is currently not being measured or collected from the client
6. Meet with Darian
- Project files deadline discussed
- Visualisation updates discussed
- doc: Update requirement 29 to include numerical value
- doc: Revise requirements to eliminate unused/unnecessary reqs
- Production deployment risks discussed
